ingestion/src/metadata/ingestion/source/database/bigquery/metadata.py

```diff
@@ -84,6 +84,13 @@ def get_columns(bq_schema):
         }
         try:
             if field.policy_tags:
+                col_obj["taxonomy"] = (
+                    PolicyTagManagerClient()
+                    .get_taxonomy(
+                        name=field.policy_tags.names[0].split("/policyTags/")[0]
```

Comment:
Can we add some checks to avoid errors in case of `field.policy_tags.names[0]` is none or `field.policy_tags.names[0].split("/policyTags/")` empty?

Reply:
yes, makes sense, let me add validations here so things don't go breaking. Thanks 🙌